### PR TITLE
Instead of h263 video freezing on reciever end, implemented fix to sh…

### PR DESCRIPTION
**Pull request by zackm0571**
…ow sideways video. Better fix coming soon

### CODE REVIEW

File: src/org/linphone/VideoCallFragment.java [hunk 1]
```diff
@@ -62,8 +62,31 @@
...
+			if (params.getUsedVideoCodec().toString().contains("H263")) {
+
+				LinphoneManager.getLc().setDeviceRotation(90);
+				LinphoneManager.getLc().updateCall(call, null);
+				viewId = R.layout.video_h263;
+
+			}
+		}
+
+		catch(NullPointerException e){
```
Comment (shareefalis) [inline src/org/linphone/VideoCallFragment.java]: NO, NEVER DO THAT!  Handle it with a if ( something != null)

File: src/org/linphone/LinphoneLauncherActivity.java [hunk 2]
```diff
@@ -32,9 +32,9 @@
 
 import org.linphone.R;
 
-/*import io.fabric.sdk.android.Fabric;
-import com.crashlytics.android.Crashlytics;
-import com.crashlytics.android.ndk.CrashlyticsNdk;*/
+//import io.fabric.sdk.android.Fabric;
+//import com.crashlytics.android.Crashlytics;
+//import com.crashlytics.android.ndk.CrashlyticsNdk;
```
Comment (shareefalis) [inline src/org/linphone/LinphoneLauncherActivity.java:38]: revert the file

File: src/org/linphone/LinphoneActivity.java [hunk 3]
```diff
@@ -991,6 +992,8 @@ public void onOrientationChanged(final int o) {
 				return;
 			}
 
+
```
Comment (shareefalis) [inline src/org/linphone/LinphoneActivity.java]: revert the file

File: src/org/linphone/VideoCallFragment.java [hunk 4]
```diff
@@ -101,6 +124,8 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
 
 		fixZOrder(mVideoView, mCaptureView);
 
+
```
Comment (shareefalis) [inline src/org/linphone/VideoCallFragment.java]: Revert the whitespaces.

File: src/org/linphone/VideoCallFragment.java (hunk 1 above)
Comment (zackm0571) [inline src/org/linphone/VideoCallFragment.java]: My bad! This was at the end of the day and we just wanted something to work. I'll add some checks and remove the try / catch right now.

File: src/org/linphone/VideoCallFragment.java [hunk 5]
```diff
@@ -77,23 +82,23 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
-					RelativeLayout.LayoutParams lp = (RelativeLayout.LayoutParams)mCaptureView.getLayoutParams();
-					lp.addRule(RelativeLayout.ALIGN_PARENT_BOTTOM, 0); // Clears the rule, as there is no removeRule until API 17.
-					lp.addRule(RelativeLayout.ALIGN_PARENT_RIGHT, 0);
-					int left = lp.leftMargin + (x - dx);
-					int top = lp.topMargin + (y - dy);
-					lp.leftMargin = left;
-					lp.topMargin = top;
-					view.setLayoutParams(lp);
-					break;
+					case MotionEvent.ACTION_DOWN:
```
Comment (shareefalis) [inline src/org/linphone/VideoCallFragment.java]: revert the whitespace.

File: src/org/linphone/VideoCallFragment.java [hunk 6]
```diff
@@ -147,6 +158,57 @@ public boolean onTouch(View v, MotionEvent event) {
 		return view;
 	}
 
+	//Check to see if current call is using an H263 video codec, if so, adjust the video for portrait
+	private boolean isH263(){
```
Comment (shareefalis) [inline src/org/linphone/VideoCallFragment.java:159]: use void unless i am missing something

File: src/org/linphone/LinphoneActivity.java [hunk 7]
```diff
@@ -35,6 +35,7 @@
 import org.linphone.core.LinphoneCall.State;
 import org.linphone.core.LinphoneCallLog;
 import org.linphone.core.LinphoneCallLog.CallStatus;
+
```
Comment (ianblenke) [inline src/org/linphone/LinphoneActivity.java]: Whitespace.

File: src/org/linphone/LinphoneActivity.java [hunk 8]
```diff
@@ -1008,6 +1009,7 @@ else if (o < 225)
 			int rotation = (360 - degrees) % 360;
 			LinphoneCore lc = LinphoneManager.getLcIfManagerNotDestroyedOrNull();
 			if (lc != null) {
+
```
Comment (ianblenke) [inline src/org/linphone/LinphoneActivity.java]: Whitespace.

File: src/org/linphone/LinphoneActivity.java [hunk 9]
```diff
@@ -1277,8 +1279,7 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
 		return super.onKeyDown(keyCode, event);
 	}
 }
-
 interface ContactPicked {
 	void setAddresGoToDialerAndCall(String number, String name, Uri photo);
 	void goToDialer();
-}
+}
```
Comment (ianblenke) [inline src/org/linphone/LinphoneActivity.java]: Just revert this file. You've added a trailing LF at the end of the file. It's easier just to revert this file.

File: src/org/linphone/LinphoneLauncherActivity.java [hunk 10]
```diff
@@ -52,7 +52,7 @@
 	protected void onCreate(Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 
-        //Fabric.with(this, new Crashlytics(), new CrashlyticsNdk());
+     //   Fabric.with(this, new Crashlytics(), new CrashlyticsNdk());
```
Comment (ianblenke) [inline src/org/linphone/LinphoneLauncherActivity.java:55]: Whitespace.

**Comment by ianblenke** [comment]
+1 LGTM

**Pull request merged**